Add nightly ok label workflow (#32876)
demisto/content: Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_13.md

#### Playbooks

##### Azure - Network Security Group Remediation

> - Added the *SubscriptionID* and *ResourceGroup* optional inputs in case the subscription ID and/or resource group needs to be specified.

Reviewer: @ShirleyDenkberg to review
```suggestion
- Added the *SubscriptionID* and *ResourceGroup* optional inputs in case the Subscription ID, or the Resource Group needs to be specified.
```

> - Add the *remediatedFlag* and *remediatedReason* outputs to supply information on whether the remediation action was successful and additional information for the parent playbook.

Reviewer: ```suggestion
- Added the *remediatedFlag* and *remediatedReason* outputs to supply informationwhether the remediation action was successful, and additional information for the parent playbook.
```

> - Fixed an issue with not being able to detect all offending rules.

Reviewer: Is this what you meant?
```suggestion
- Fixed an issue where not all all offending rules were detected.
```
